docker/README.md

```diff
@@ -154,7 +154,7 @@ Docker has two available versions: Community Edition (CE) and Enterprise Edition
 
 1. Verify the installation:
 
-        docker run --runtime=nvidia --rm nvidia/cuda nvidia-smi
```

Comment:
If I understand it correctly, this change is not strictly related to the update to focal but more about the version of docker in the user system. I think we can update it as you did in the PR since modern docker uses the native nvidia integration.

Reply:
This is correct. I decided to update to reflect the docs found at https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html#docker to hopefully reduce confusion.

I'm not too familiar with what is actually going on in docker/nvidia-runtime internals so maybe this change in CLI arguments is required.